Update BigQuery Data Graph Setup Guide (#7285)
segmentio/segment-docs: src/unify/data-graph/setup-guides/BigQuery-setup.md

11. Copy all the content in the JSON file you created in the previous step, and save it for Step 5. 

> ## Step 2: Create a dataset for Segment to store checkpoint tables
> **Segment requires write access to this dataset for internal bookkeeping and to store checkpoint tables for the queries that are executed. Therefore, Segment recommends creating a new dataset for this purpose.** 
> 
> > info ""
> > Segment recommends creating a new dataset for the Data Graph.
> > If you choose to use an existing dataset that has also been used for [Segment Reverse ETL](/docs/connections/reverse-etl/), you must follow the [additional instructions](#update-user-access-for-segment-reverse-etl-catalog) to update user access for the Segment Reverse ETL catalog.
> 
> Navigate to the BigQuery SQL editor and create a dataset that will be used by Segment. 
> 
> ```
> CREATE SCHEMA IF NOT EXISTS `__segment_reverse_etl`;
> GRANT `roles/bigquery.dataEditor` ON SCHEMA `__segment_reverse_etl` TO "serviceAccount:<YOUR SERVICE ACCOUNT EMAIL>";
> ```
> 

Reviewer: ```suggestion
## Step 2: Create a dataset for Segment to store checkpoint tables
Create a new dataset as Segment requires write access to the dataset for internal bookkeeping and to store checkpoint tables for the queries that are executed. 

Segment recommends you to create a new dataset for the Data Graph. If you choose to use an existing dataset that has also been used for [Segment Reverse ETL](/docs/connections/reverse-etl/), you must follow the [additional instructions](/docs/unify/data-graph/setup-guides/bigquery-setup/#update-user-access-for-segment-reverse-etl-dataset) to update user access for the Segment Reverse ETL catalog.

To create your dataset, navigate to the BigQuery SQL editor and create a dataset that will be used by Segment. 

```
CREATE SCHEMA IF NOT EXISTS `__segment_reverse_etl`;
GRANT `roles/bigquery.dataEditor` ON SCHEMA `__segment_reverse_etl` TO "serviceAccount:<YOUR SERVICE ACCOUNT EMAIL>";
```

```
